feature/platform/transceiver/tests/zr_inventory_test/zr_inventory_test.go

```diff
@@ -0,0 +1,150 @@
+package zr_inventory_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/featureprofiles/internal/samplestream"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygot/ygot"
+)
+
+const (
+	dp16QAM          = 1
+	samplingInterval = 10 * time.Second
+	timeout          = 10 * time.Minute
+	waitInterval     = 30 * time.Second
+)
+
+const (
+	frequency         = 193100000
+	targetOutputPower = -10
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+func interfaceConfig(t *testing.T, dut1 *ondatra.DUTDevice, dp *ondatra.Port, frequency uint64, targetOutputPower float64) {
```

Comment:
nit, can we call this configInterface() instead as this reads better because this "configures the interface" as opposed to "return interface config."

Reply:
Done